datasets/sk/rpvs/crawler.py

```diff
@@ -1,14 +1,182 @@
-import zipfile
+from zavod import Context, helpers as h
 
-from zavod import Context
-from zavod.shed.bods import parse_bods_fh
+# from zavod.shed.bods import parse_bods_fh
+BASE_URL = "https://rpvs.gov.sk/opendatav2"
+ENTITY_DETAILS_ENDPOINT = (
+    f"{BASE_URL}/PartneriVerejnehoSektora/{{id}}?$expand=Partner,PravnaForma,Adresa"
+)
+PARTNER_DETAILS_ENDPOINT = f"{BASE_URL}/Partneri/{{id}}?$expand=Vymaz,Pokuta,OverenieIdentifikacieKUV,konecniUzivateliaVyhod,verejniFunkcionari,kvalifikovanePodnety"
+# TODO: \u0161
+BENEFICIAL_OWNERS_ENDPOINT = (
+    f"{BASE_URL}/KonecniUzivateliaVyhod/{{id}}?$expand=Partner,PravnaForma,Adresa"
+)
+PUBLIC_OFFICIALS_ENDPOINT = f"{BASE_URL}/VerejniFunkcionari/{{id}}"
 
+TOTAL_COUNT = "https://rpvs.gov.sk/opendatav2/PartneriVerejnehoSektora/$count"
```

Comment:
(nit) TOTAL_COUNT is unused. Maybe we could use that as a way to terminate the crawler?